exporters/metric/prometheus/prometheus.go

```diff
@@ -135,42 +135,49 @@ func NewRawExporter(config Config) (*Exporter, error) {
 // 	http.HandleFunc("/metrics", hf)
 // 	defer pipeline.Stop()
 // 	... Done
-func InstallNewPipeline(config Config) (*push.Controller, http.HandlerFunc, error) {
-	controller, hf, err := NewExportPipeline(config, time.Minute)
+func InstallNewPipeline(config Config, options ...push.Option) (*push.Controller, *Exporter, error) {
+	controller, exp, err := NewExportPipeline(config, options...)
 	if err != nil {
-		return controller, hf, err
+		return controller, exp, err
 	}
+	controller.Start()
 	global.SetMeterProvider(controller)
-	return controller, hf, err
+	return controller, exp, err
 }
 
 // NewExportPipeline sets up a complete export pipeline with the recommended setup,
 // chaining a NewRawExporter into the recommended selectors and integrators.
-func NewExportPipeline(config Config, period time.Duration) (*push.Controller, http.HandlerFunc, error) {
-	selector := simple.NewWithHistogramDistribution(config.DefaultHistogramBoundaries)
+//
+// The returned Controller implements `metric.Provider`.  The controller is
```

Comment:
This will conflict with changes included in master.

```suggestion
// The returned Controller contains an implementation of `metric.Provider`.  The controller is
```

Reply:
Done.